admin/differ/Differ.scss

```diff
@@ -7,9 +7,13 @@
 
 .xh-admin-diff-detail {
   width: 700px;
+  max-height: 80vh;
   table {
+    display: flex;
     width: 100%;
```

Comment:
This `width: 100%` is redundant and can be removed - the width of the table is already space filling due to being in a `vframe`